spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonEagerLoadProperties.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.netflix.ribbon;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.List;
+
+/*
+ * Configuration Properties to indicate which Ribbon configurations 
+ * should be eagerly loaded up
+ * 
+ * @author Biju Kunjummen
+ */
+@ConfigurationProperties(prefix = "ribbon.eager-load")
```

Comment:
Should the namespace be `ribbon`?  Since these properties are specific to Spring Cloud Ribbon, should they be `spring.cloud.ribbon`?  Not sure on the conventions we have used in the past.  @spencergibb, what do you think?